db/schema.rb

```diff
@@ -115,6 +115,17 @@
     t.index ["seller_id"], name: "index_menu_items_on_seller_id"
   end
 
+  create_table "open_hours", force: :cascade do |t|
```

Comment:
Curious, how would you model a store that's open 24-hours?

Reply:
I think we could store it as 12am (day n) to 12am (day n+1) and have a check on the frontend to show "Open 24 Hours"